File: Server/CQS/DTOs/ColonyDto.cs
using System;

namespace Server.CQS.DTOs
{
    public class ColonyDto
    {
        public Guid Id { get; set; }
        public bool IsActive { get; set; }
        public string CreatedBy { get; set; }
        public long CreatedDate { get; set; }
        public string LastModifiedBy { get; set; }
        public long LastModifiedDate { get; set; }

        public Guid HostId { get; set; }

        public Guid AreaId { get; set; }

        public int HiveNumber { get; set; }

        public string ColonyNumber { get; set; }
        public string ColonySource { get; set; }
        public string QueenType { get; set; }

        public string Markings { get; set; }

        public string GeneticBreed { get; set; }

        public string InstallationType { get; set; }

        public string AdditionalInfo { get; set; }

        public long InstallDate { get; set; }

        public string HiveType { get; set; }
        public string BroodChamberType { get; set; }

        public bool QueenExclude { get; set; }
    }
}

File: App/app/src/main/java/com/example/apiverte_colony_management/DTOs/ColonyDTO.java
package com.example.apiverte_colony_management.DTOs;

import java.time.LocalDateTime;
import java.util.UUID;

public class ColonyDTO {
    public UUID Id;

    public String Name;

    public Boolean IsActive;

    public String CreatedBy;

    public long CreatedDate;
    public String LastModifiedBy;
    public long LastModifiedDate;

    public UUID HostId;

    public UUID AreaId;

    public int HiveNumber;

    public String ColonyNumber;
    public String ColonySource;
    public String QueenType;

    public String Markings;

    public String GeneticBreed;

    public String InstallationType;

    public String AdditionalInfo;

    public long InstallDate;

    public String HiveType;
    public String BroodChamberType;

    public Boolean QueenExclude;
}
